rollup.config.js

```diff
@@ -46,4 +56,3 @@ export default [
     context: null,
   },
 ];
-9;
```

Comment:
Why is the EOF line removed?

Reply:
There was a weird `9;` added on the end line (i think it was introduced by someone by mistake). My intention was to just remove the unwanted code from the last line. Anyway, I can't figure out why git doesn't pick the new line at the end of the file cause VSCode adds it on file save.

Reply:
Okay, thanks for cleaning it up!